plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/ValidationUtilsTest.java

```diff
@@ -53,40 +53,40 @@
       LINE_WITH_WHITESPACE + PROJECT_ID;
   
   private DocumentLocation location = new DocumentLocation(2, 13);
-  private ElementProblem element =
-      new AppEngineBlacklistElement("application", location, 3);
-  private ArrayList<ElementProblem> blacklist = new ArrayList<>(Arrays.asList(element));
+  private ElementProblem problem =
+      new AppEngineDeprecatedElement("application", location, 3);
+  private ArrayList<ElementProblem> problems = new ArrayList<>(Arrays.asList(problem));
```

Comment:
Is `ArrayList` explicitly required? Otherwise, could be just `Arrays.asList(problem)`. And can this be `final`?

Reply:
this does seem to need to be an array list, at least until we modify the model code too.